Sources/SwiftDocC/Indexing/Navigator/NavigatorItem.swift

```diff
@@ -151,26 +151,6 @@ public final class NavigatorItem: Serializable, Codable, Equatable, CustomString
         return data
     }
     
-    // MARK: - Equatable
-    
-    public static func == (lhs: NavigatorItem, rhs: NavigatorItem) -> Bool {
-        return lhs.pageType == rhs.pageType &&
-            lhs.languageID == rhs.languageID &&
-            lhs.title == rhs.title &&
-            lhs.platformMask == rhs.platformMask &&
-            lhs.availabilityID == rhs.availabilityID
-    }
-    
-    // MARK: - Hasher
-    
-    public func hash(into hasher: inout Hasher) {
-        hasher.combine(pageType)
-        hasher.combine(languageID)
-        hasher.combine(title)
-        hasher.combine(platformMask)
-        hasher.combine(availabilityID)
-    }
```

Comment:
Neither of these consider the `path`, `usrIdentifier`, or `icon` which is _probably_  a bug that would be avoided by relying on the default synthesized implementations.